.golangci.yml

```diff
@@ -71,7 +71,7 @@ linters-settings:
     #  2. you use go >= 1.10
     #  3. you do repeated runs (false for CI) or cache $GOPATH/pkg or `go env GOCACHE` dir in CI.
     use-installed-packages: false
-  golint:
+  revive:
```

Comment:
Any reason we are switching? Is revive running a superset of linting rules?

Reply:
golint is deprecated (repository is archived, and latest version of golangci-lint uses revive to replace it). revive has the same feature as golint, not going to break anything I think